README.md

```diff
@@ -5,7 +5,8 @@
 ## Contributing
 
 1. Write a typings definition and upload to GitHub - here's an [example using debug](https://github.com/typings/typed-debug/blob/master/typings.json)
-2. Make a pull request, using [semver](http://semver.org/) to specify which version the typing is valid for - here's [debug again](https://github.com/typings/registry/blob/master/npm/debug.json)
+2. Use [semver](http://semver.org/) to specify which version the typing is valid for - here's [debug again](https://github.com/typings/registry/blob/master/npm/debug.json)
+3. Run `npm test` and check that the registry is valid, once passed do a pull request
 3. Once the pull request is merged, it is used as the canonical reference - `typings install debug`
```

Comment:
That's just how markdown is parsed. It only has to start with a number, they could be 1. 1. 1. 1. and it'll still render correctly because they become `<li>` elements which can't actually set the number. This is the raw Markdown though, if you could just quickly correct that :smile: